service/src/main/resources/templates/ehr_link_set_template.mustache

```diff
@@ -0,0 +1,49 @@
+<component typeCode="COMP"{{#isNested}} contextConductionInd="true"{{/isNested}}>
+    <LinkSet classCode="OBS" moodCode="EVN">
+        <id root="{{linkSetId}}" />
+        <effectiveTime>
+            {{#effectiveTimeLow}}<low value="{{effectiveTimeLow}}"/>{{/effectiveTimeLow}}{{^effectiveTimeLow}}<low nullFlavor="UNK"/>{{/effectiveTimeLow}}
+            {{#effectiveTimeHigh}}<high value="{{effectiveTimeHigh}}"/>{{/effectiveTimeHigh}}{{^effectiveTimeHigh}}<high nullFlavor="UNK"/>{{/effectiveTimeHigh}}
```

Comment:
Confirm if missing high value should be null flavour unknown or omitted. Mapping spreadsheet does not specify using <high nullFlavor="UNK"/> here

Reply:
Will remove, confirmed with @dmcavenue1